cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/DefaultNamespaceAdmin.java

```diff
@@ -406,6 +407,8 @@ public synchronized void updateProperties(Id.Namespace namespaceId, NamespaceMet
     }
 
     nsStore.update(builder.build());
+    // refresh the cache with new meta
+    namespaceMetaCache.refresh(namespaceId.toEntityId());
```

Comment:
@chtyim I have this in AppFabricServiceRuntimeModule
```
bind(NamespaceAdmin.class).to(DefaultNamespaceAdmin.class).in(Scopes.SINGLETON);    
bind(NamespaceQueryAdmin.class).to(DefaultNamespaceQueryAdmin.class).in(Scopes.SINGLETON);
```
Since NamespaceQueryAdmin is being binded in Singleton scope NamespaceAdmin which extends NamesapceQueryAdmin will use the same instance. Please correct me, if my understanding is wrong.